pkg/webhook/webhook.go

```diff
@@ -130,6 +130,12 @@ func (wh *webhook) validate(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionRespo
 		response.Allowed = true
 		return response
 	}
+	// ignore the Delete request if requested resource doesn't exists
```

Comment:
comment can be like
`ignore the Delete request of PVC if resource name is empty which can happen as part of cleanup process of namespace`

Reply:
Updated the comments